addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -537,3 +537,34 @@ a.no-decoration {
 .o_shadow_animation {
     box-shadow: 0 0 5px 10px rgba(240,8,0,0.4)!important;
 }
+
+/* product recently viewed snippet */
+
+.o-product-card {
```

Comment:
_ instead of -

but this name seems really generic for something specific you do in a small snippet ? Is the css even useful ? Should not we reused the style of the /shop page ?